ui/src/chatbot.tsx

```diff
@@ -70,6 +71,11 @@ const
       marginBottom: 7.5,
       transform: 'translateX(-50%)',
       width: 180
+    },
+    feedback: {
```

Comment:
Why does this need to be absolutely positioned?

Reply:
A leftover from right placed buttons implementation.